<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>dlg_register</class>
 <widget class="QDialog" name="dlg_register">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>454</width>
    <height>440</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QLabel#lb_title{font:40px '黑体';}
QLabel#lb_username,QLabel#lb_password{font:20px '楷体';}
QLineEdit{border-radius:4px;min-height:28px;border:1px solid black}
QPushButton{border-radius:4px;background: #409eff; color:white;font -size:14px;}
QWidget#widget{background:white;}</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_2">
   <item row="0" column="0">
    <widget class="QLabel" name="label">
     <property name="text">
      <string>姓名</string>
     </property>
     <property name="name" stdset="0">
      <string>lb</string>
     </property>
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="QLineEdit" name="le_nickname">
     <property name="name" stdset="0">
      <string>le</string>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QLabel" name="label_2">
     <property name="text">
      <string>学院</string>
     </property>
     <property name="name" stdset="0">
      <string>lb</string>
     </property>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="QLineEdit" name="le_department">
     <property name="name" stdset="0">
      <string>le</string>
     </property>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="QLabel" name="label_3">
     <property name="text">
      <string>年级</string>
     </property>
     <property name="name" stdset="0">
      <string>lb</string>
     </property>
    </widget>
   </item>
   <item row="2" column="1">
    <widget class="QLineEdit" name="le_grade">
     <property name="name" stdset="0">
      <string>le</string>
     </property>
    </widget>
   </item>
   <item row="3" column="0">
    <widget class="QLabel" name="label_4">
     <property name="text">
      <string>用户名</string>
     </property>
     <property name="name" stdset="0">
      <string>lb</string>
     </property>
    </widget>
   </item>
   <item row="3" column="1">
    <widget class="QLineEdit" name="le_username">
     <property name="name" stdset="0">
      <string>le</string>
     </property>
    </widget>
   </item>
   <item row="4" column="0">
    <widget class="QLabel" name="label_5">
     <property name="text">
      <string>密码</string>
     </property>
     <property name="name" stdset="0">
      <string>lb</string>
     </property>
    </widget>
   </item>
   <item row="4" column="1">
    <widget class="QLineEdit" name="le_password">
     <property name="name" stdset="0">
      <string>le</string>
     </property>
    </widget>
   </item>
   <item row="5" column="0" colspan="2">
    <widget class="QWidget" name="widget" native="true">
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>60</height>
      </size>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <item row="0" column="0">
       <widget class="QPushButton" name="btn_register">
        <property name="maximumSize">
         <size>
          <width>120</width>
          <height>30</height>
         </size>
        </property>
        <property name="text">
         <string>注册</string>
        </property>
        <property name="name" stdset="0">
         <string>btn</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QPushButton" name="btn_login">
        <property name="maximumSize">
         <size>
          <width>120</width>
          <height>30</height>
         </size>
        </property>
        <property name="text">
         <string>登陆</string>
        </property>
        <property name="name" stdset="0">
         <string>btn</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
